src/test/groovy/org/whaka/mock/EventCollectorTest.groovy

```diff
@@ -0,0 +1,84 @@
+package org.whaka.mock
+
+import spock.lang.Specification
+
+import java.util.function.BiConsumer
+import java.util.function.Predicate
+
+/**
+ * @author gdzabaev
+ */
+class EventCollectorTest extends Specification {
+
+    private BiConsumer<Listener, Integer> method = {l,e -> l.event(e)}
+    private Predicate<Integer> filter = Mock()
+    private EventCollector.EventHandler<Integer> filterEH = Mock()
+    private EventCollector<Listener, Integer> collector = EventCollector.create(Listener.class, method, filter, filterEH)
+    private Listener l = collector.getTarget()
```

Comment:
Please create all required for test instances and mocks directly in test methods under the `given:` label. It improves readability and understandability of tests.

Thanks.